app/services/meeting_notifications_service.rb

```diff
@@ -19,49 +18,59 @@ def self.handle_members(args = {})
     new(args).handle_members
   end
 
-  # rubocop:disable MethodLength
   def handle_members
     members.each do |member|
-      next if member.user_id == current_user.id
+      next if member.id == current_user.id
 
-      data = notification_data
-      Notification.create(
-        user_id: member.user_id,
-        uniqueid: uniqueid,
-        data: data
-      )
-      notifications = Notification.where(user_id: member.user_id)
-                                  .order('created_at ASC').all
-      Pusher["private-#{member.user_id}"].trigger(
-        'new_notification',
-        notifications: notifications
-      )
-      NotificationMailer.notification_email(member.user_id, data).deliver_now
+      send_member_notification(member, notification_data)
     end
   end
 
   private
 
-  def notification_data
-    group = Group.where(id: meeting.group_id).first.name
-    if type == 'remove_meeting'
-      return {
-        user: current_user.name,
-        group_id: meeting.group_id,
-        group: group,
-        typename: meeting.name,
-        type: type,
-        uniqueid: uniqueid
-      }.to_json
-    end
+  def get_notifications(member, data)
+    Notification.create(
```

Comment:
what happens if the notification is not created? I think we need to cater for that

Reply:
Thanks for the important call out! Updated!